cloud/pkg/dynamiccontroller/application/eventhandler.go

```diff
@@ -76,6 +82,8 @@ func (c *handlerCenter) AddListener(s *SelectorListener) error {
 }
 
 func (c *handlerCenter) DeleteListener(s *SelectorListener) {
+	c.handlerLock.Lock()
+	defer c.handlerLock.Unlock()
```

Comment:
It will be more efficient to not use the function `defer` in a simple process function.

Reply:
done